assets/css/styles.css

```diff
@@ -1,3 +1,3 @@
a screen and (max-width: 1023px){.grid-columns .use-11-columns.offset-11{grid-column:span 12}}.grid-columns .use-12-columns{grid-column:span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns{grid-column:span 12}}.grid-columns .use-12-columns.offset-2{grid-column:2/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-2{grid-column:span 12}}.grid-columns .use-12-columns.offset-3{grid-column:3/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-3{grid-column:span 12}}.grid-columns .use-12-columns.offset-4{grid-column:4/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-4{grid-column:span 12}}.grid-columns .use-12-columns.offset-5{grid-column:5/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-5{grid-column:span 12}}.grid-columns .use-12-columns.offset-6{grid-column:6/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-6{grid-column:span 12}}.grid-columns .use-12-columns.offset-7{grid-column:7/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-7{grid-column:span 12}}.grid-columns .use-12-columns.offset-8{grid-column:8/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-8{grid-column:span 12}}.grid-columns .use-12-columns.offset-9{grid-column:9/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-9{grid-column:span 12}}.grid-columns .use-12-columns.offset-10{grid-column:10/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-10{grid-column:span 12}}.grid-columns .use-12-columns.offset-11{grid-column:11/span 12}@media screen and (max-width: 1023px){.grid-columns .use-12-columns.offset-11{grid-column:span 12}}@media screen and (max-width: 768px){.main-content .container{padding-left:1rem;padding-right:1rem}}@media screen and (max-width: 768px){.main-content .container.mobile-no-padding{padding-left:0;padding-right:0}}.page .main-content,.single .main-content{padding-top:0;padding-bottom:2rem}.page .main-content .entry-page-content,.single .main-content .entry-page-content{padding-top:2.5rem;padding-bottom:3rem}.page-template-page-home .main-content{padding-top:0;padding-bottom:0}.page-template-page-home .homepage-sidebars{margin-top:3rem}.main-header .navbar .navbar-menu .menu-item.current-menu-item{color:#000}.main-header .navbar .navbar-menu .menu-item.current-menu-item>a{color:#000}.main-header .navbar .navbar-menu .menu-item.current-menu-ancestor>a{color:#000}.push-up{margin-top:-4rem}.terms-list h6{margin-bottom:.5rem}.terms-list .button{margin-right:1rem;margin-bottom:.5rem}.error404 .page-header .container{padding-top:10rem;padding-bottom:10rem}.error404 .page-header .container h4{margin-bottom:1rem}.text-format p{margin-bottom:1.5rem}.text-format ul{padding-left:1.5rem;list-style:disc}.text-format ul li{margin-bottom:1rem}.text-format ol{list-style:n
```

Comment:
I'm in favor of removing this mini reset line altogether.

Reply:
it is a comment for the styles that follow. removing it breaks the display

Reply:
Yes, it seems to be the minified Vocabulary styles inlined into this file. We need to figure out how to develop the Vocabulary styles in a single location.